docs/howto/operate/new-hub/index.md

```diff
@@ -1,25 +1,75 @@
+(new-hub)=
 # Add a new hub
 
-This section describes steps around adding new hubs to the 2i2c JupyterHub federation.
+This section describes steps around adding hubs to the 2i2c JupyterHub federation.
 
-## General overview
+## Infrastructure that is needed for a hub
 
-We use a GitHub Issue Template to keep track of the to-do items in creating a new hub.
-The best way to understand the information and steps needed for this process is in that template.
-Below is a short overview of the steps in the process.
+There are three kinds of infrastructure needed to add a new hub:
 
-1. [Create a new GitHub issue for the hub](https://github.com/2i2c-org/pilot-hubs/issues/new?assignees=&labels=type%3A+hub&template=new-hub.md&title=New+Hub%3A+%5BHub+name%5D).
-   The following steps are major sections described in the template.
-2. Collect information about the hub that helps us deploy it in the proper manner (see the issue template for explanations about the information we need).
-3. Create the hub via appending a new hub entry in the appropriate cluster file under
-   `config/hubs` file (see [](/topic/config.md) for more information).
-4. Start following [team process](team-process) around operating the hubs.
+- **A Kubernetes cluster**.
+  Deploying a Kubernetes cluster is specific to the cloud provider. For hubs that do not need to use their own cloud credits, or otherwise are fine running on a cloud project that is not owned by their institution, we can deploy hubs on an already-running Kubernetes Cluster.
+  For hubs that require their own cluster, we'll need to set it up on our own with Terraform.
+  To do so, see [new-cluster].
+- **Support infrastructure**.
+  This is a collection of services that run on a Kubernetes Cluster and help us in running and monitoring things.
+  There is one set of services running per **cluster**, not per hub.
+  This includes things like Grafana, NFS server provisioners, etc.
+- **JupyterHubs**.
+  When a cluster is up and running, we may then deploy JupyterHubs on top of it using the JupyterHub Helm Chart.
+  This is generally controlled via configuration in the `pilot-hubs/` repository, and GitHub actions that automatically deploy new hubs (or modify existing ones) on a cluster.
```

Comment:
> This is generally controlled via configuration in the `pilot-hubs/` repository

For me, this is giving the idea the other pieces you mentioned above are not controlled by the pilot-hubs repo and that is not the case...

Reply:
OK I've tried to clarify this a bit